src/main/kotlin/com/devfloor/untitled/articlehashtag/application/ArticleHashtagService.kt

```diff
@@ -0,0 +1,16 @@
+package com.devfloor.untitled.articlehashtag.application
+
+import com.devfloor.untitled.article.domain.Article
+import com.devfloor.untitled.articlehashtag.domain.ArticleHashtag
+import com.devfloor.untitled.articlehashtag.domain.ArticleHashtagRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class ArticleHashtagService(
+    private val repository: ArticleHashtagRepository
+) {
+    @Transactional(readOnly = true)
+    fun showAllByArticle(article: Article): List<ArticleHashtag> =
+        repository.findAllByArticle(article)
```

Comment:
이렇게 표현식 쓸 경우 반환타입 명시 안 해도 되니까 없애도 될 듯?

```suggestion
    fun showAllByArticle(article: Article) = repository.findAllByArticle(article)
```

Reply:
여기도 위에 써놓은거처럼 그냥 다 명시하는게 어때?
가독성 측면에서(물론 주관적인거지만) 명시하는게 훨씬 좋지않나??